src/main/java/org/assertj/core/api/AbstractLocalDateAssert.java

```diff
@@ -264,6 +267,26 @@ public SELF isNotEqualTo(String localDateAsString) {
     return isNotEqualTo(parse(localDateAsString));
   }
 
+  /**
+   * Same assertion as {@link #isEqualTo(Object)} (where Object is expected to be {@link javax.xml.datatype.XMLGregorianCalendar})
+   * but here you pass {@link javax.xml.datatype.XMLGregorianCalendar}
+   * <p>
+   * Example :
+   * <pre><code class='java'> // use XMLGregorianCalendar  as-is without conversion
+   * assertThat(parse("2000-01-01")).isEqualTo(xmlGregorianCalendar);</code></pre>
+   *
```

Comment:
add a note about time zone with an example showing the assertion behavior when the local time zone is not the same as the provided `XMLGregorianCalendar`.